data/required_tasks.yml

```diff
@@ -29,6 +29,21 @@ pipeline-required-tasks:
         - inspect-image
         - summary
   docker:
+    - effective_on: "2025-03-01T00:00:00Z"
```

Comment:
Just in case the PRs to add the new tasks (from https://issues.redhat.com/browse/EC-1063 ) are not done in time, we could push this date out further.

Reply:
(In a future PR I mean.)